core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksStore.java

```diff
@@ -229,4 +237,44 @@ public synchronized void close() {
     mDbOpts.close();
     LOG.info("Closed store at {}", mDbPath);
   }
+
+  // helper function to load RockDB configuration options based on property key configurations.
+  static Optional<BlockBasedTableConfig> checkSetTableConfig(
```

Comment:
Are these property keys subject to change? I mean do you need to pass those as parameters?

Reply:
I think this is just a helper function to check whether the property keys have been assigned.

Reply:
Yes it loads the property keys into the rocks db config if the property keys are set. It is there to avoid duplicate code for each of the Inode, edge, block meta, and block location tables.